### ISSUE TITLE
Completion Window doesn't resize while you are typing a name

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#21413]
This completion window seems to have a new feature where it's showing the name of the thing you are typing, it doesn't resize though as you type a name larger than the Window:
![image](https://user-images.githubusercontent.com/1103906/29154568-b6d3e884-7dd8-11e7-922e-2c1d84c49a7a.png)

**Comment by Pilchie** [comment]
This would be an issue in the core platform.  Tagging @AmadeusW and @olegtk to see if they have a dupe in VSTS.

**Comment by olegtk** [comment]
Yes, it's completion platform issue. How did you get into this state, @davkean ? 
The feature you are talking about is called suggestion completion mode and it's toggled by Ctrl+Alt+Space.
In this mode we don't grow completion popup indeed. Not sure if it's by design or not. Only Roslyn supports this mode btw.  I'll file a bug to track it anyway.

**Comment by davkean** [comment]
@olegtk This still occurs - did you end up filing a bug? We now enter suggestion mode in almost all naming contexts making this more prevalent.

**Comment by olegtk** [comment]
not sure. @AmadeusW ?

**Comment by CyrusNajmabadi** [comment]
I don't repro this.  It resizes for me:

<img width="1387" height="450" alt="Image" src="https://github.com/user-attachments/assets/000d739c-86be-481d-bb69-58c829199794" />
